CHANGELOG.md

```diff
@@ -17,6 +17,16 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [7.54.8] - 2024-07-22
+### Added
+- The method `client.functions.schedules.retrieve` now accepts the missing parameter `ignore_unknown_ids` as well
+  as retrieving multiple schedules at once.
+- The method `client.functions.schedules.create` now supports creating using a `FunctionScheduleWrite` object.
+- 
+### Changed
+- When creating a new function schedules without specifying `description`, the default values are now
+  correctly set to `None` instead of `""`.
```

Comment:
```suggestion
- When creating a new function schedule without specifying `description`, the default values are now
  correctly set to `None` instead of `""`.
```